infrastructure/code/bundle.go

```diff
@@ -186,42 +183,42 @@ func (b *Bundle) retrieveAnalysis(ctx context.Context) ([]snyk.Issue, error) {
 	}
 }
 
-func (b *Bundle) addCodeActions(ctx context.Context, issues []snyk.Issue) ([]snyk.Issue, error) {
+func (b *Bundle) addCodeActions(ctx context.Context, issues []snyk.Issue) {
 	method := "addCodeActions"
 
 	autoFixEnabled := getCodeSettings().isAutofixEnabled.Get()
 	learnEnabled := config.CurrentConfig().IsSnykLearnCodeActionsEnabled()
 	log.Debug().Str("method", method).Msg("Autofix is enabled: " + strconv.FormatBool(autoFixEnabled))
 	log.Debug().Str("method", method).Msg("Snyk Learn is enabled: " + strconv.FormatBool(learnEnabled))
 
-	if autoFixEnabled || learnEnabled {
-		for i := range issues {
-			if autoFixEnabled {
-				// We only allow the issues whose file extensions are supported by the
-				// backend.
-				supported, err := b.isAutofixSupportedForExtension(ctx, issues[i].AffectedFilePath)
-				if err != nil {
-					return []snyk.Issue{}, err
-				}
+	if !autoFixEnabled && !learnEnabled {
+		log.Trace().Msg("Autofix | Snyk Learn code actions are disabled, not adding code actions")
+		return
+	}
 
+	for i := range issues {
+		if autoFixEnabled {
+			// We only allow the issues whose file extensions are supported by the
+			// backend.
+			supported, err := b.isAutofixSupportedForExtension(ctx, issues[i].AffectedFilePath)
+			if err != nil {
+				log.Error().Str("method", method).Msg("Failed to check if autofix is supported for extension.")
```

Comment:
Would you please add the error to the log with `.Err(err)`? Also, it maybe beneficial to have it captured with error reporter (if not already done in the called method).

Reply:
Good catch, added it ✅